Core/Frameworks/Baikal/Model/Config.php

```diff
@@ -82,34 +80,32 @@ protected function parseConfig($sString) {
 
                 $sValue = $aMatches[1][0];    # first capture (.*?), first occurence (anyway, we asserted that there's only one)
                 switch ($aConstant["type"]) {
-                    case "string": {
+                    case "string":
```

Comment:
It takes the `{}` block out of switch statements. I could try to find and disable that fixer. But it seems OK to "fix" it - not too many lines of diff.